Source/Orts.Simulation/Simulation/RollingStocks/SubSystems/Brakes/MSTS/AirSinglePipe.cs

```diff
@@ -274,6 +349,7 @@ public override void Parse(string lowercasetoken, STFReader stf)
         {
             switch (lowercasetoken)
             {
+                case "wagon(brakecylinderpressureformaxbrakeforce":
```

Comment:
This is a mistake I made once that made me question the original syntax. I'm not sure if anyone has released rolling stock with this 'mistake' in it (nothing I own seems to make this error) so it may be over protective of the userbase to allow it.